compiler-plugin-tests/src/test/resources/ballerina_sources/validator_tests/01_valid_return_types/service.bal

```diff
@@ -20,6 +20,8 @@ type Person record {
     string name;
 };
 
+type Error distinct error;
```

Comment:
Can't we use `graphql:Error` type instead of defining a new type?

Reply:
This is just a compiler plugin test. The semantics are the same for both types. I'll add another test to be safe anyway.